samples/official/camera-tagging/README.md

```diff
@@ -25,7 +25,48 @@ This module is currently only supports the ARM32 and AMD64 platforms and it does
 1. Sign in to your Azure Container Registry by entering the following command in the Visual Studio Code integrated terminal (replace <REGISTRY_USER_NAME>, <REGISTRY_PASSWORD>, and <REGISTRY_NAME> to your container registry values set in the .env file).
     - `docker login -u <REGISTRY_USER_NAME> -p <REGISTRY_PASSWORD> <REGISTRY_NAME>.azurecr.io`
 
-1. Fill in the environment variables found in deployment.template.json 
+1. Fill in the environment variables found in deployment.template.json. If targeting the AI Dev Kit device use: **deployment.peabody.template.json**
+
+1. Select your target architecture. Open the command palette and search for Azure IoT Edge: Set Default Target Platform for Edge Solution, or select the shortcut icon in the side bar at the bottom of the window. In the promoted window, select arm32v7.
+
+1. Right-click on deployment.template.json and select the **Build and Push IoT Edge Solution** command to generate a new deployment.json file in the config folder, build a module image, and push the image to the specified ACR repository. If targeting the AI Dev Kit device use: **deployment.peabody.template.json**
+    > Note: Some red warnings "Unknown host QEMU_IFLA type: ##" and "qemu: Unsupported syscall: ##" displayed during the building process can be ignored.
+
+1. Right-click on config/deployment.json, select **Create Deployment for Single Device**, and choose the targeted IoT Edge device to deploy the container image.
+
+1. You'll find troubleshooting steps at <https://visionaidevkitsupport.azurewebsites.net/>.
+
+### ARM64
+
+1. ARM64 support is in preview, to add the new os/arch option to your IoT Edge tools in VS code, update setings settings (JSON) by adding:
```

Comment:
I don t think that users actually need to edit this paltform settings instead they should edit the module.json file in the module folder of your solution.  Current module.json file is missing the arm64 option.  That way it would the same steps as for other platforms and we can make the instructions shorter and clearer. Thanks!

Reply:
README.md updated.
Tested with the latest version of the Azure IoT Tools and the arm64v8 option it is only needed in the module.json and not in the settings.json as well.